packages/clerk-js/src/utils/captcha/retrieveCaptchaInfo.ts

```diff
@@ -11,11 +11,7 @@ export const retrieveCaptchaInfo = (clerk: Clerk) => {
     captchaWidgetType: _environment ? _environment.displayConfig.captchaWidgetType : null,
     captchaProvider,
     captchaPublicKeyInvisible: _environment ? _environment.displayConfig.captchaPublicKeyInvisible : null,
-    canUseCaptcha: _environment
-      ? _environment.userSettings.signUp.captcha_enabled &&
-        clerk.isStandardBrowser &&
-        clerk.instanceType === 'production'
-      : null,
+    canUseCaptcha: _environment ? _environment.userSettings.signUp.captcha_enabled && clerk.isStandardBrowser : null,
```

Comment:
@issuedat Can you update the PR description with an explanation on this one ?
I remember that this was done on purpose in the first iteration, can you give context on has changed ?

Reply:
I've updated the description with some more context! Let me know if that helps ☺️